packages/zoe/src/types.js

```diff
@@ -43,5 +43,6 @@
  */
 
 /**
+ * @template [OfferResult]
```

Comment:
where is this used?

Is it straightforward to demonstrate how it's used in a test?

Reply:
It's used in `packages/zoe/src/zoeService/types.js#Offer` as the type for `zoe.offer`'s invitation, and before introducing this optional template parameter, the type of the first arg to `zoe.offer` failed and was silently replaced by `any`.  The producing code is in `packages/zoe/src/contractFacet/types.js#MakeInvitation`, which will propagate the offer result types.

The reason this didn't cause a type error is that Zoe extensively uses the `@callback` tag, which TypeScript currently doesn't report `@param` type syntax or semantic problems.  It's a known issue in TS, but I don't have the link handy.

Reply:
Ok. I'm familiar with the ts bug.